test/integration/stub-schema.json

```diff
@@ -1881,6 +1943,19 @@
                 "required": true
               }
             },
+            {
+              "name": "submitter_primary_diagnosis_id",
```

Comment:
I think we can actually delete version 2.0 & 3.0  and the diff section. Would make it easier to update only one schema. But this doesn't have to happen in this PR.

Reply:
I tought Wajiha based on it in schema tests? will check

Reply:
Its fine this was just me thinking out lound. I think we might want to do a refactor here, cause the `migration-schema` stuff should really be in a folder called `dictionary` and rename schema to dictionary

Reply:
I don't think it has to be this PR

Reply:
yeah agree will do something about it probably when I do the schema migration related bug